CMakeLists.txt

```diff
@@ -97,18 +116,18 @@ endif (NOT CMAKE_CXX_FLAGS)
 if (NOT CMAKE_BUILD_TYPE)
     # Use debug mode if building in dbg or debug directory.
     get_filename_component(_cmpvar ${CMAKE_BINARY_DIR} NAME)
-    if(_cmpvar STREQUAL "dbg" OR _cmpvar STREQUAL "debug")
+    if("_cmpvar" STREQUAL "dbg" OR "_cmpvar" STREQUAL "debug")
```

Comment:
Unintended.  Either screwed up the merge or failed to get the latest 
copy.  I did

>  1019  git pull upstream master
>  1021  git checkout -b LibrarySoVersioning

to make the feature branch starting with the latest from github 
(upstream is casacore/casacore); at least that was my intent.


On 07/01/2015 01:39 PM, Julian Taylor wrote:
>
> In CMakeLists.txt 
> <https://github.com/casacore/casacore/pull/144#discussion_r33716153>:
>
> > @@ -97,18 +116,18 @@ endif (NOT CMAKE_CXX_FLAGS)
> >  if (NOT CMAKE_BUILD_TYPE)
> >      # Use debug mode if building in dbg or debug directory.
> >      get_filename_component(_cmpvar ${CMAKE_BINARY_DIR} NAME)
> > -    if(_cmpvar STREQUAL "dbg" OR _cmpvar STREQUAL "debug")
> > +    if("_cmpvar" STREQUAL "dbg" OR "_cmpvar" STREQUAL "debug")
>
> these quotes where removed in gh-110 
> <https://github.com/casacore/casacore/pull/110>, are they needed again 
> or is this unintended?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/casacore/casacore/pull/144/files#r33716153>.Web 
> Bug from 
> https://github.com/notifications/beacon/AGPGtOvYyUV3Wn9xnODg0y9HXdbbdZ-2ks5oZDlugaJpZM4FQHXV.gif
>

-- 
Jim Jacobs

phone: 835-7235
office: 301